README.md

```diff
@@ -32,6 +32,10 @@ make go-install
 stellar-disbursement-platform --help
 ```
 
+> [!WARNING]  
+> We're in the process of releasing a new multi-tenant version of the SDP. Develop branch currently contains the release candidate of multi-tenant SDP. 
+> For the stable single-tenant version, please use the latest `v1.x.x` tag.
```

Comment:
do we want to link the release tags here? eg https://github.com/stellar/stellar-disbursement-platform-backend/releases

Reply:
addressed in 1b836460e43e6fc0fc48184e4626f9186ebec034